src/modules.cc

```diff
@@ -17,8 +17,11 @@ static bool file_exists(const char *fname) noexcept {
 }
 
 static void lua_init(lua_State *L) {
-  const auto user_dir = std::string(RimeGetUserDataDir());
-  const auto shared_dir = std::string(RimeGetSharedDataDir());
+  // path::string() returns native encoding on Windows
```

Comment:
`RimeGetUserDataDir`, `RimeGetSharedDataDir` are deprecated.
They still work, but less efficient. Because the internal data has changed from `string` to `path`, they will do extra conversion and allocating string in static variable.